elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
I'm a little nervous about the arrangement of a child firing parent-specific events to manipulate the state of an assumed parent. This pattern tightly couples what would otherwise be two atomic elements. What do you think?